packages/react-components/react-components/etc/react-components.unstable.api.md

```diff
@@ -368,6 +368,22 @@ export { scrollToItemStatic }
 
 export { ScrollToItemStaticParams }
 
+export { TeachingBubble }
```

Comment:
`/unstable` within react-components is deprecated and we cannot modify it. please undo these changes